charts/airbyte/templates/scheduler/deployment.yaml

```diff
@@ -192,11 +192,46 @@ spec:
         {{- if .Values.scheduler.resources }}
         resources: {{- toYaml .Values.scheduler.resources | nindent 10 }}
         {{- end }}
+        {{- if .Values.scheduler.containerSecurityContext }}
+        securityContext: {{- toYaml .Values.scheduler.containerSecurityContext | nindent 10 }}
+        {{- end }}
+        {{- if .Values.scheduler.livenessProbe.enabled }}
+        livenessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - -ec
+            - grep -qa airbyte.scheduler.app.SchedulerApp /proc/1/cmdline
```

Comment:
Could you please explain what the liveness probes commands are checking exactly? 🙏

Reply:
Sure! Since the scheduler is not opening any port, the only thing we can do is to ensure that the entrypoint was the `airbyte.scheduler.app.SchedulerApp` java class and not another random command. In `/proc/1/cmdline` we can find the command executed for the entrypoint (`pid=1`) since there is no `ps` command inside the container.

This is basically needed because probes are a requisite for some admission controllers (e.g.: [Kyverno `require_probes` policy](https://github.com/kyverno/policies/blob/main/best-practices/require_probes/require_probes.yaml)).